### DISCUSSION TITLE
Welcome to Serenity Discussions!

### DISCUSSION THREAD

**Discussion by YajanaRao** [YajanaRao/Serenity#82] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MansourM** [comment]
Hi, 
your project looks awesome so as somebody who is learning react native, i wanted to look into your code to help me better understand react native and get familiar with react native best code practices, and also to get ideas to use in my own projects,
but unfortunately i got stuck i the first step :) i cant build/run it, I tried several things and searched for solutions but non seem to work, i installed all dependencies in root and packages also did some other tweaking but my IDE reports Errors on all imports (it have something to do with typescript but i cant figure it out) and when u try building the project i get a gradle error.
I know its very basic stuff but it can get overwhelming for newer devs like me.
we would be very grateful if you can add a getting started/installation section so we can get started.

**Reply by YajanaRao** [reply]
Hi,
Getting started is not in sync with the latest code. Checkout branch exp/db code where I have the latest changes done. It should be easy to setup and run the project. Let me know if you see issues in that branch as well

**Reply by YajanaRao** [reply]
https://github.com/YajanaRao/Serenity/tree/exp/db

**Reply by YajanaRao** [reply]
Checkout the telegram channel or discord channel if you have an account.

**Comment by YajanaRao** [comment]
Adding getting started documentation to todo on priority.
